presto-openapi/src/main/resources/rest_function_server.yaml

```diff
@@ -250,6 +250,17 @@ components:
           type: string
           description: The version of the function. This version shall be maintained by REST server for any change in the function.
           example: "1"
+        executionEndpoint:
+          type: string
+          format: uri
+          pattern: "^https?://"
+          description: |
+            Base URL of the execution server where this function should be executed.
```

Comment:
Is this execution model defined? Will a driver send multiple batches in parallel (i.e., multiplexed) over the same connection, or will the driver and server follow a sequential/wait model where each batch is acknowledged before sending?

Reply:
But that will need operator changes for the driver to send batches and process async responses?

Reply:
These already exist--`RemoteProjectOperator` in Java, and analogous C++ operators.

Reply:
Nice, another think we should consider was to add a limit to the number of rows sent at an operator level (for eg if these functions are sent to ML models) etc

Reply:
I guess that can be implemented in the server itself too in a way

Reply:
I checked but velox doest seem to have a remote project operator. Is that support left?